ansible/roles/common/tasks/moodle.yml

```diff
@@ -18,13 +18,17 @@
 - name: Clone moodle
   ansible.builtin.git:
     repo: https://github.com/moodle/moodle.git
-    dest: /var/moodle
+    dest: "{{ moodle_root_path }}"
     version: MOODLE_401_STABLE
```

Comment:
I think it is a good idee to make the moodle version a variable and store it in the vars.yaml